pysd/py_backend/builder.py

```diff
@@ -47,20 +48,20 @@ def add(cls, module, function=None):
             setattr(cls, f"_{module}", True)
 
     @classmethod
-    def get_header(cls, outfile):
+    def get_header(cls, outfile, root=False):
         """
         Returns the importing information to print in the model file
```

Comment:
Is the `root` argument here something that only takes a true/false value, or could it also take a path? looks like it's only used in the boolean sense, but if that's the long-term plan for it, might make sense to give it a different name?

Reply:
I will update the name with `force_root` and add the documentation to the method. If this argument is True the `_root` is added in the model. If the parameter is false, the `_root` will be added only if the model has `External` objects.